nengo/neurons.py

```diff
@@ -89,6 +89,38 @@ def gain_bias(self, max_rates, intercepts):
 
         return gain, bias
 
+    def max_rates_intercepts(self, gain, bias):
+        """Compute the max_rates and intercepts corresponding to the given
+        gain and bias.
+
+        Note that this default implementation is very slow! Whenever possible,
+        subclasses should override this with a neuron-specific implementation.
+
+        Parameters
+        ----------
+        gain : ndarray(dtype=float64)
+            Gain associated with each neuron. Sometimes denoted alpha.
+        bias : ndarray(dtype=float64)
+            Bias current associated with each neuron.
+
+        Returns
+        -------
+        max_rates : ndarray(dtype=float64)
+            Maximum firing rates of neurons.
+        intercepts : ndarray(dtype=float64)
+            X-intercepts of neurons.
+        """
+
+        max_rates = self.rates(np.ones_like(gain), gain, bias)
+
+        x_range = np.linspace(-1, 1, 101)
+        rates = np.asarray([self.rates(np.ones_like(gain) * x, gain, bias)
+                            for x in x_range])
+        last_zeros = np.maximum(np.argmax(rates > 1e-16, axis=0) - 1, 0)
```

Comment:
`finfo.tiny` seems a bit too small to me (e.g., ~1e-300 for float64).  We could just say `rates > 0` if we only want to know if the value is positive, the idea here is that there is some point that the firing rate is "low enough" that we'll call it 0.  But yeah, I don't have a good reason for picking `1e-16`, other than that is the cutoff we were already using in `gain_bias`.  We could adjust both of those if we think that's too high/low.